outputs.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "addresses" {
+  description = "List of address values managed by this module (e.g. [\"1.2.3.4\"])"
+  value       = ["${local.ip_addresses}"]
+}
+
+output "names" {
+  description = "List of address resource names managed by this module (e.g. [\"gusw1-dev-fooapp-fe-0001-a-0001-ip\"])"
+  value       = ["${local.ip_names}"]
+}
+
+output "gcp_dns_fqdns" {
```

Comment:
Do we need the `gcp_` prefix to indicate what's going on here? If we need to prefix something to add context, can we indicate that this is Cloud DNS?

Reply:
This was leftover from when the module handled other forms of DNS registration, I think it makes sense to remove it